Common/UnitDefinitions/HeatFlux.json

```diff
@@ -116,6 +116,30 @@
           "Abbreviations": [ "kcal/h·m²" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundForcePerFootSecond",
+      "PluralName": "PoundsForcePerFootSecond",
+      "FromUnitToBaseFunc": "x*1.459390293720636e1",
+      "FromBaseToUnitFunc": "x/1.459390293720636e1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbf/(ft·s)", "lb/(ft·s)" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerSecondCubed",
+      "PluralName": "PoundsPerSecondCubed",
+      "FromUnitToBaseFunc": "x*4.5359237e-1",
+      "FromBaseToUnitFunc": "x/4.5359237e-1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lbm/s³", "lb/s³" ]
```

Comment:
Is `lbm` the preferred notation over `lb`?

Reply:
I will switch these. I hate imperial units :)

Reply:
Haha, me too 😆 Come on world, let's standardize on something more sane than your own foot and a big, heavy stone you happened to have lying around.